ResearchKit/ActiveTasks/ORKTowerOfHanoiStepViewController.m

```diff
@@ -57,7 +57,7 @@ @implementation ORKTowerOfHanoiViewController {
     NSDate *_firstMoveDate;
 }
 
-#pragma Mark -- UIViewController
+#pragma mark -- UIViewController
```

Comment:
For the separator line to work it needs to be one dash character (`-`) instead of two (`--`). I should have mentioned this explicitly in my earlier comment, sorry!